### PR TITLE
Issue 6209

### PR DESCRIPTION
**Pull request by Khrisefzm**
Add pading and width in diferent sections
Add correct link in the button

### CODE REVIEW

**Comment by Khrisefzm** [comment]
Issue link:
https://github.com/orgs/breatheco-de/projects/8/views/1?filterQuery=assignee%3A%40me&pane=issue&itemId=49859365

File: src/data/page/job-guarantee.es.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ two_column_left:
       color: "white"
       background: "black"
       hover_color: "white"
-      path: "#top"
+      path: "/es/aplica"
```
Comment (tommygonzaleza) [inline src/data/page/job-guarantee.es.yml:47]: Dejalo con `#top`

File: src/data/page/job-guarantee.us.yml [hunk 2]
```diff
@@ -49,7 +49,7 @@ two_column_left:
       color: "white"
       background: "black"
       hover_color: "white"
-      path: "#top"
+      path: "/us/apply"
```
Comment (tommygonzaleza) [inline src/data/page/job-guarantee.us.yml:52]: Dejalo con `#top`

File: src/templates/job-guarantee.js [hunk 3]
```diff
@@ -379,9 +380,19 @@ const JobGuarantee = ({ data, pageContext, yml }) => {
...
 
-      <WeTrust we_trust={yml.we_trust_section} />
+      <Div padding="0 10px" padding_tablet="0 90px" padding_lg="0 130px">
+        <WeTrust we_trust={yml.we_trust_section} />
+      </Div>
 
-      <Div display="block" margin="40px 0px">
+      <Div
+        display="block"
+        maxWidth="1695px"
```
Comment (tommygonzaleza) [inline src/templates/job-guarantee.js:389]: El max-width de nuestra pagina es 1366px

**Review by tommygonzaleza — CHANGES_REQUESTED** [review]

**Review by tommygonzaleza — APPROVED** [review]

**Review by tommygonzaleza — APPROVED** [review]

**Review by alesanchezr — APPROVED** [review]

**Pull request merged**